src/cmi_docx/paragraph.py

```diff
@@ -9,6 +9,7 @@
 from docx.text import run as docx_run
 
 from cmi_docx import run, styles
+from cmi_docx.comment import CommentPreserver
```

Comment:
Minor styling: could you import the module rather than the class for consistency with the rest of the codebase..